docs/guide/resources/synthetic_dataset_generation.md

```diff
@@ -43,6 +43,17 @@ config = {
 }
 ```
 
+If you want to provide label descriptions to further help the LLM understand what each label means, you can change the label list to a dictionary as shown below:
+
+```python
```

Comment:
lets add this to the default way of doing dataset generation because I think this would be important to get reasonable results to start with